swagger/coordinator.yml

```diff
@@ -0,0 +1,263 @@
+        400:
+          description: PET Error
+  /participant/sum2:
+    post:
+      tags:
+        - Coordinator
+      description: PET Step 12 (send "update" message)
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Sum2Message"
+      responses:
+        200:
+          description: OK
+        400:
+          description: PET Error
+  /global_model:
+    get:
+      tags:
+        - Aggregator
+      description: Download the global model
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+                format: byte
+        404:
+          description: model not found
+components:
+  schemas:
+    SumMessage:
+      description: Sum message
+      type: object
+      required:
+        - message
+      properties:
+        message:
+          type: object
+          required:
+            - round_box
+            - sum_box
+          properties:
+            round_box:
+              type: string
+              format: byte
+              minLength: 65
+              maxLength: 65
+            sum_box:
+              type: string
+              format: byte
+    UpdateMessage:
+      description: Update message
+      type: object
+      required:
+        - message
+        - mask_seed
+      properties:
+        message:
+          type: object
+          required:
+            - round_box
+            - update_box
+          properties:
+            round_box:
+              type: string
+              format: byte
+              minLength: 65
+              maxLength: 65
+            update_box:
+              type: string
+              format: byte
+        mask_seed:
+          description: "The encrypted seed."
+          type: string
+          format: byte
+          minLength: 32
+          maxLength: 32
+    Sum2Message:
+      description: Sum2 message
+      type: object
+      required:
+        - message
+        - mask_url
+      properties:
+        message:
+          type: object
+          required:
+            - round_box
+            - sum2_box
+          properties:
+            round_box:
+              type: string
+              format: byte
+              minLength: 65
+              maxLength: 65
+            sum2_box:
+              type: string
+              format: byte
+        mask_url:
+          description: The URL that points to the global mask. The aggregator can use this URL to fetch the global mask.
+          type: string
+          format: url
+    PublishRoundMessage:
+      type: object
+      properties:
+        coord_pk:
+          description: The public key of the coordinator.
+          type: string
+          format: byte
+          minLength: 32
+          maxLength: 32
+        sum:
+          description: The sum threshold of the current round.
```

Comment:
The description seems a bit unclear to me. Maybe we could directly adapt this from the PET protocol document:

> fraction of participants to be selected in this round for computing the unmasking sum

And something similar for `update`